binrw_derive/src/parser/attrs.rs

```diff
@@ -17,8 +17,8 @@ pub(crate) type Default = kw::default;
 pub(crate) type DerefNow = kw::deref_now;
 pub(crate) type If = MetaList<Token![if], Expr>;
 pub(crate) type Ignore = kw::ignore;
-pub(crate) type Import = MetaList<kw::import, IdentPatType>;
-pub(crate) type ImportTuple = MetaValue<kw::import_tuple, IdentPatType>;
+pub(crate) type Import = MetaList<kw::import, NamedImport>;
+pub(crate) type ImportTuple = MetaValue<kw::import_tuple, NamedImport>;
```

Comment:
should import_tuple be able to specify a default value? how would such a value get used?

as an aside, @csnover, iirc this is the reason there's a split between args and args_tuple -- there's a split between import and import_tuple, because i didn't/don't know of a good way to do `import =` and still specify both the name and the type. so it became `import_tuple(name: type)` and i just mirrored it with args, becoming `args_tuple(...)`